config/resources/publicatie-domain.json

```diff
@@ -49,6 +49,10 @@
         "modified": {
           "type": "datetime",
           "predicate": "dct:modified"
+        },
+        "status-modified": {
+          "type": "datetime",
+          "predicate": "dossier:statusAanpassingsDatum"
```

Comment:
- `dossier` is een namespace die niet onder ons beheer valt. Hier mag je niet zelf predicaten in beginnen definiëren
- het model bevat al een `publication-status-change` resource met een `started-at` om een status wijziging voor te stellen. Kan deze niet gebruikt worden?

Reply:
Kheb de commit gerevert :D aangezien dit al bestond